projects/packages/my-jetpack/_inc/components/product-cards-section/types.ts

```diff
@@ -0,0 +1,6 @@
+import type { FC } from 'react';
+
+export type ProductCardType = FC< {
```

Comment:
Idea for name: `ProductCardComponent`/`ProductCardFC`? I like the unification!

Reply:
`ProductCardComponent` sounds better to me, thanks!